desktop-src/direct3dhlsl_current/hlsl-built-ins-functions_math_logic.md

```diff
@@ -0,0 +1,3332 @@
+# Math and Logic Operations
+
+### abs
```

Comment:
If all these are going to be described on the same page, we at least need TOC nodes for navigation.  It's also visually hard to parse and see where one operation ends and another begins.  I wish the content under an operation was all indented or something, so it would be visually obvious.

Reply:
Could parameter, return value and type tables be merged into one table by simply adding the description to the type table?

I imagine this would need to be done with a script, otherwise it's a lot of mechanical labor.

Reply:
I could write a simple C# program with regular expressions.  It won't be too complicated.